target/linux/ath79/nand/base-files/lib/upgrade/platform.sh

```diff
@@ -13,6 +13,12 @@ platform_do_upgrade() {
 	local board=$(board_name)
 
 	case "$board" in
+	aerohive,hiveap-121|\
+	domywifi,dw33d|\
+	netgear,wndr4300|\
+	zyxel,nbg6716)
+		nand_do_upgrade "$1"
+		;;
```

Comment:
I've made nand_do_upgrade default here, so you can drop changes to this file entirely.